src/aapns/models.py

```diff
@@ -68,9 +69,18 @@ def get_dict(self) -> Dict[str, Any]:
         return alert
 
 
+@unique
+class PushType(Enum):
+    alert = "alert"
+    background = "background"
+
+
 @attr.s
 class Notification:
     alert: Alert = attr.ib(validator=attr.validators.instance_of(Alert))
+    push_type: PushType = attr.ib(
```

Comment:
An attr.ocious class be instantiated with plain arguments: `Notification(1, 2, 3)`.
Thus, adding an argument in the middle of the attribute list is a breaking change.

IMO, here, it would be simpler to add it at the end?
Or, if this mirrors some Apple API, then bump up the major version of the package.

Reply:
Resolving b/c major version change anyway.